flatten_dict/flatten_dict.py

```diff
@@ -29,6 +31,7 @@ def flatten(d, reducer='tuple', inverse=False, enumerate_types=()):
         used to reduce.
         'tuple': The resulting key will be tuple of the original keys.
         'path': Use `os.path.join` to join keys.
+        'underscore': Use underscores to join keys.
```

Comment:
Don't forget the type above: `{'tuple', 'path', Callable}` -> `{'tuple', 'path', 'underscore', Callable}`